src/cr/cube/matrix/measure.py

```diff
@@ -551,7 +556,20 @@ class _Sums(_BaseSecondOrderMeasure):
     @lazyproperty
     def blocks(self):
         """2D array of the four 2D "blocks" making up this measure."""
-        return SumSubtotals.blocks(self._cube_measures.cube_sum.sums, self._dimensions)
+        return SumDiffSubtotals.blocks(
```

Comment:
Is this on purpose? Do we want to change Sum -> SumDiff? Doesn't seem related to this PR.

Reply:
it is not related but i prefer to release it in this cr.cube version cause we don't want sum subtotals but we wanna have the subtotal differences.

Reply:
just avoid 2 releases .. that;s the only reason